memory-db/src/lib.rs

```diff
@@ -238,23 +242,25 @@ pub fn legacy_prefixed_key<H: KeyHasher>(key: &H::Out, prefix: Prefix) -> Vec<u8
 	prefixed_key
 }
 
-impl<H, KF, T> Default for MemoryDB<H, KF, T>
+impl<H, KF, T, S> Default for MemoryDB<H, KF, T, S>
 where
 	H: KeyHasher,
 	T: for<'a> From<&'a [u8]>,
 	KF: KeyFunction<H>,
+	S: BuildHasher + Default,
```

Comment:
```suggestion
impl<H, KF, T> Default for MemoryDB<H, KF, T, RandomState>
where
H: KeyHasher,
T: for<'a> From<&'a [u8]>,
KF: KeyFunction<H>,
```

This way you can just use `default()` without declaring the random state. Similar to `HashMap` we should then provide a function `with_random_state`